contentcuration/contentcuration/templates/base.html

```diff
@@ -24,8 +25,11 @@
         } catch (error) { /* Page does not require front-end translations */ }
       </script>
 			<script src="{% url 'js_reverse' %}" type="text/javascript"></script>
-			<script type="text/javascript" src="{% static 'js/bundles/common.js' %}"></script>
-			<script type="text/javascript" src="{% static 'js/bundles/base.js' %}"></script>
+
+			{% render_bundle 'common' %}
```

Comment:
@jayoshih @rtibbles Add a chunk to the template to specify a bundle, with `base` a default.

From [webpack's docs](https://webpack.js.org/configuration/entry-context/#entry):
> Simple rule: one entry point per HTML page. SPA: one entry point, MPA: multiple entry points.

Because webpack leans away from relying on implicit global variables, things get complicated when there are 2 bundles on a page (like we were doing before). Doubly so when we rely on so many of our libraries' [side effects](https://github.com/learningequality/studio/pull/582/files#diff-a9736753be95233c0f19acafe655629bR7).

A "downside" I'm seeing with this approach, though, is that everything is in the `<head>` - this leads to slightly slower perceived load times for some pages, `channel_edit` being the noticeable one for me. I think we can fix this down the line with intelligently-placed async imports. We could also move this down to the body? I dunno. I got the sense that this was done intentionally, so that the navbar would display before everything else. Would love your guys' thoughts.